docs/reference/ml/anomaly-detection/apis/get-snapshot.asciidoc

```diff
@@ -164,6 +168,10 @@ reclaim space.
 `model_bytes_memory_limit`:::
 (long) The upper limit for memory usage, checked on increasing values.
 
+`peak_model_bytes`:::
+(long) The maximum memory usage of the model.
```

Comment:
This sounds a bit like the maximum that **can** be used. How about we rephrase to `The highest recorded value for the model memory usage`?

Reply:
```suggestion
(long) The highest recorded value for the model memory usage.
```